### PR TITLE
Converts "Logos" page into "Brand Guidelines," updates assets

### PR DESCRIPTION
**Pull request by jeffdaley**
## What it does

- Converts the "logos" page into "brand guidelines"
- Updates the logo asset files
- Adds link to new brand guidelines pdf

### CODE REVIEW

**Comment by wifelette** [comment]
Annnd... don't merge just yet :p I want @MelSumner to take a look but also @jeffdaley is going to add a commit with some tweaks we discussed.

**Comment by jeffdaley** [comment]
Tertiary logos added to the Guidelines screen and PDF.

One thing I couldn't figure out is how to change the footer link from "Logos" to "Brand Guidelines." My change in the `_footer.erb` file didn't work.

Also, should we change the file and url from `/logos` to `/brand-guidelines`?

File: source/_header.html.erb [hunk 1]
```diff
@@ -55,30 +55,30 @@
 				<li role="separator" class="divider"></li> %>
        <li><a href="/team">The Team</a></li>
        <li role="separator" class="divider"></li>
-       <li><a href="/logos">Logos</a></li>
+       <li><a href="/logos">Brand Guidelines</a></li>
```
Comment (MelSumner) [inline source/_header.html.erb]: Not blocking, maybe pedantic- "Branding" instead of "Brand Guidelines" ?

Comment (jeffdaley) [reply source/_header.html.erb]: I like that. Especially since we have "community guidelines."

**Comment by MelSumner** [comment]
@jeffdaley that's because the footer is coming from ember-styleguide - @locks can you confirm?

**Comment by sivakumar-kailasam** [comment]
The middleman parts of the website in this repo uses https://github.com/ember-learn/glimmer-styleguide. Everything else uses https://github.com/ember-learn/ember-styleguide

**Pull request merged**